pymc/distributions/continuous.py

```diff
@@ -1662,7 +1662,7 @@ class AsymmetricLaplace(Continuous):
     rv_op = asymmetriclaplace
 
     @classmethod
-    def dist(cls, b, kappa, mu=0, *args, **kwargs):
+    def dist(cls, kappa, mu, b, *args, **kwargs):
```

Comment:
Looking at the Laplace, should it be this? 
```suggestion
    def dist(cls, mu, b kappa, *args, **kwargs):
```

Also need to reorder the parameters in the docstrings to reflect the new order

Reply:
As I said in the original issue, SkewNormal and StudentT both have their non-normality/non-symmetry arguments first (before location and scale), so it was unclear why `kappa` would be different.

Reply:
Sounds good. Should still reorder the docstring parameters

Reply:
Just checked and this matches scipy's [asymmetric Laplace signature](https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.laplace_asymmetric.html).  The [Half]StudentT signature (now) matches [scipy](https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.t.html) as well (no default value for `nu`).